build.gradle.kts

```diff
@@ -1,14 +1,12 @@
 import org.gradle.api.tasks.wrapper.Wrapper.DistributionType.ALL
 
+@Suppress("DSL_SCOPE_VIOLATION") // false positive for `libs` in IntelliJ
 plugins {
-    kotlin("jvm")
-    kotlin("plugin.serialization")
-    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.12.1"
-    id("org.jetbrains.dokka")
+    org.jetbrains.dokka
 
     signing
     `maven-publish`
-    id("io.codearte.nexus-staging") version "0.30.0"
+    alias(libs.plugins.nexusStaging)
```

Comment:
Disable plugins for the root project (kord), like:
```kts
    org.jetbrains.dokka apply false
    signing apply false
    `maven-publish` apply false
    alias(libs.plugins.nexusStaging) apply false
```

Reply:
Dokka has to be applied to the root project so the `dokkaHtmlMultiModule` task that we use for generating docs at https://kordlib.github.io/kord is added.
I guess `signing` and `maven-publish` can be removed altogether from the root project.
I don't know what the nexus staging plugin does, so I'm not sure about that one.

Reply:
I removed the nexus staging plugin, we never used the tasks. The Nexus GUI is used instead. Right @HopeBaron?